src/devices/Common/Iot/Device/Multiplexing/tests/OutputSegmentTests.cs

```diff
@@ -29,6 +29,24 @@ public void SegmentValuesWritePinValues()
                 segment[2] == 0 &&
                 segment[3] == 1);
         }
+        
+        [Fact]
+        public void SegmentValuesWriteByteOffset()
+        {
+            // segment length > written value
+            VirtualOutputSegment segment = new(12);
+            segment.Write(0b_1001_0110);
+
+            Assert.True(
```

Comment:
Can you instead do:

```csharp
Assert.Equal(0, segment[0]);
// ... 
```

also, shouldn't Write write to 8 rightmost bit rather than leftmost? (segment[4]..segment[11])

Reply:
per offline conversation right most means least significant which implies index=0